src/test/java/edu/cornell/kfs/fp/batch/service/impl/CreateAccountingDocumentServiceImplTest.java

```diff
@@ -427,6 +429,12 @@ public void testServiceError() throws Exception {
         boolean results = createAccountingDocumentService.createAccountingDocumentsFromXml();
         assertFalse("When there is a problem calling services, the job should fail", results);
     }
+    
+    @Test
+    public void testLoadSingleFileWithSingleDIDocumentWithBadAttachment() throws Exception {

```

Comment:
Is the attachment bad or the URI? would it be better to make this method name more specific to what the problem is?

Reply:
how about changing it to testLoadSingleFileWithSingleDIDocumentWithBadAttachmentUrl() ?

Reply:
sounds good, thank you